docs/requirements.txt

```diff
@@ -0,0 +1,7 @@
+mkdocs==1.5.0
+mkdocs-material==8.2.6
+mkdocs-spellcheck==0.2.1
+markdown_include==0.6.0
```

Comment:
`markdown_include` wasn't in the `Makefile` [above](https://github.com/argoproj/argo-workflows/pull/12360/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L715). We don't use it either as it's not in our `mkdocs.yml` [as a plugin](https://github.com/cmacmackin/markdown-include/tree/v0.8.1?tab=readme-ov-file#usage) and this syntax isn't used in the docs afaik

```suggestion
```